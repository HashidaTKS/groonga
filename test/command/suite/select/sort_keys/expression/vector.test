table_create Values TABLE_NO_KEY
column_create Values numbers COLUMN_VECTOR Int32

load --table Values
[
{"numbers": [127, 128, 129]},
{"numbers": [126, 255]},
{"numbers": [128, -254]}
]

select Values --sort_keys 'numbers[1]' --output_columns numbers
