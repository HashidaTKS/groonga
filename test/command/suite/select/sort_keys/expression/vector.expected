table_create Values TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Values numbers COLUMN_VECTOR Int32
[[0,0.0,0.0],true]
load --table Values
[
{"numbers": [127, 128, 129]},
{"numbers": [126, 255]},
{"numbers": [128, -254]}
]
[[0,0.0,0.0],3]
select Values --sort_keys 'numbers[1]' --output_columns numbers
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        3
      ],
      [
        [
          "numbers",
          "Int32"
        ]
      ],
      [
        [
          128,
          -254
        ]
      ],
      [
        [
          127,
          128,
          129
        ]
      ],
      [
        [
          126,
          255
        ]
      ]
    ]
  ]
]
